### PR TITLE
[PB-1645] fix: Check if directories (root and temp) exists on start fuseapp

### PR DESCRIPTION
**Pull request by ldandradesw**
https://inxt.atlassian.net/browse/PB-1645

### CODE REVIEW

**Review by JoanVicens — APPROVED** [review]
I've tested the app and the app still crashes on logout. But I think that this error is not related to the folder not existing but a listener in the ipc render that cannot be removed.

**Review by JoanVicens — CHANGES_REQUESTED** [review]
Without the log that the folder exists its good to go

File: src/apps/shared/fs/ensure-folder-exists.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
-import { PathLike, mkdirSync } from 'fs';
+import { PathLike, accessSync, mkdirSync, constants } from 'fs';
+import Logger from 'electron-log';
 
 export function ensureFolderExists(folder: PathLike) {
-  mkdirSync(folder, { recursive: true });
+  try {
+    accessSync(folder, constants.F_OK);
+    Logger.info(`Folder <${folder}> already exists`);
```
[the remark this answers is not in the document]
Comment (ldandradesw) [reply src/apps/shared/fs/ensure-folder-exists.ts:7]: Ready the change

**Comment by ldandradesw** [comment]
> I've tested the app and the app still crashes on logout. But I think that this error is not related to the folder not existing but a listener in the ipc render that cannot be removed.

image

I don't see the error when I test the logout process

**Review by JoanVicens — APPROVED** [review]

**Pull request merged**